2017-06-25 09:54:54,667 [Thread-9:9448] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-25 10:03:56,701 [Thread-17:8128] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 413, in login
    return login(request, **defaults)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\auth\views.py", line 49, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\auth\views.py", line 76, in login
    auth_login(request, form.get_user())
  File "C:\mypro\flxm\lib\site-packages\django\contrib\auth\__init__.py", line 110, in login
    request.session.cycle_key()
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\base.py", line 302, in cycle_key
    self.create()
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\db.py", line 49, in create
    self._session_key = self._get_new_session_key()
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\base.py", line 155, in _get_new_session_key
    if not self.exists(session_key):
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\db.py", line 45, in exists
    return self.model.objects.filter(session_key=session_key).exists()
  File "C:\mypro\flxm\lib\site-packages\django\db\models\query.py", line 651, in exists
    return self.query.has_results(using=self.db)
  File "C:\mypro\flxm\lib\site-packages\django\db\models\sql\query.py", line 501, in has_results
    return compiler.has_results()
  File "C:\mypro\flxm\lib\site-packages\django\db\models\sql\compiler.py", line 819, in has_results
    return bool(self.execute_sql(SINGLE))
  File "C:\mypro\flxm\lib\site-packages\django\db\models\sql\compiler.py", line 848, in execute_sql
    cursor.execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\mysql\base.py", line 112, in execute
    return self.cursor.execute(query, args)
  File "C:\mypro\flxm\lib\site-packages\pymysql\cursors.py", line 158, in execute
    result = self._query(query)
  File "C:\mypro\flxm\lib\site-packages\pymysql\cursors.py", line 308, in _query
    conn.query(q)
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 820, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 1002, in _read_query_result
    result.read()
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 1285, in read
    first_packet = self.connection._read_packet()
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 966, in _read_packet
    packet.check_error()
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 394, in check_error
    err.raise_mysql_exception(self._data)
  File "C:\mypro\flxm\lib\site-packages\pymysql\err.py", line 120, in raise_mysql_exception
    _check_mysql_exception(errinfo)
  File "C:\mypro\flxm\lib\site-packages\pymysql\err.py", line 112, in _check_mysql_exception
    raise errorclass(errno, errorvalue)
ProgrammingError: (1146, u"Table 'flaccdb.django_session' doesn't exist")
2017-06-25 10:06:59,296 [Thread-19:7032] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-25 10:07:15,410 [Thread-29:3980] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 413, in login
    return login(request, **defaults)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\auth\views.py", line 49, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\debug.py", line 76, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\auth\views.py", line 76, in login
    auth_login(request, form.get_user())
  File "C:\mypro\flxm\lib\site-packages\django\contrib\auth\__init__.py", line 110, in login
    request.session.cycle_key()
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\base.py", line 302, in cycle_key
    self.create()
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\db.py", line 49, in create
    self._session_key = self._get_new_session_key()
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\base.py", line 155, in _get_new_session_key
    if not self.exists(session_key):
  File "C:\mypro\flxm\lib\site-packages\django\contrib\sessions\backends\db.py", line 45, in exists
    return self.model.objects.filter(session_key=session_key).exists()
  File "C:\mypro\flxm\lib\site-packages\django\db\models\query.py", line 651, in exists
    return self.query.has_results(using=self.db)
  File "C:\mypro\flxm\lib\site-packages\django\db\models\sql\query.py", line 501, in has_results
    return compiler.has_results()
  File "C:\mypro\flxm\lib\site-packages\django\db\models\sql\compiler.py", line 819, in has_results
    return bool(self.execute_sql(SINGLE))
  File "C:\mypro\flxm\lib\site-packages\django\db\models\sql\compiler.py", line 848, in execute_sql
    cursor.execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "C:\mypro\flxm\lib\site-packages\django\db\backends\mysql\base.py", line 112, in execute
    return self.cursor.execute(query, args)
  File "C:\mypro\flxm\lib\site-packages\pymysql\cursors.py", line 158, in execute
    result = self._query(query)
  File "C:\mypro\flxm\lib\site-packages\pymysql\cursors.py", line 308, in _query
    conn.query(q)
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 820, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 1002, in _read_query_result
    result.read()
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 1285, in read
    first_packet = self.connection._read_packet()
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 966, in _read_packet
    packet.check_error()
  File "C:\mypro\flxm\lib\site-packages\pymysql\connections.py", line 394, in check_error
    err.raise_mysql_exception(self._data)
  File "C:\mypro\flxm\lib\site-packages\pymysql\err.py", line 120, in raise_mysql_exception
    _check_mysql_exception(errinfo)
  File "C:\mypro\flxm\lib\site-packages\pymysql\err.py", line 112, in _check_mysql_exception
    raise errorclass(errno, errorvalue)
ProgrammingError: (1146, u"Table 'flaccdb.django_session' doesn't exist")
2017-06-25 17:43:17,838 [Thread-3:9008] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-25 17:43:28,729 [Thread-9:8876] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-25 17:43:33,594 [Thread-10:9912] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-06-25 17:46:59,759 [Thread-2:8880] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-06-25 21:14:29,121 [Thread-2:944] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-06-25 22:00:27,474 [Thread-2:8348] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-06-25 22:01:39,476 [Thread-4:9916] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-06-26 19:32:47,368 [Thread-4:6696] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-26 19:32:58,477 [Thread-5:3108] [django.request:182] [base:get_response] [WARNING]- Not Found: /ADMIN
2017-06-26 19:34:07,036 [Thread-9:7296] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-26 20:57:45,842 [Thread-4:9776] [django.request:182] [base:get_response] [WARNING]- Not Found: /fl_kt/static/js/kindeditor-4.1.10/kindeditor-min.js
2017-06-26 20:57:46,005 [Thread-7:9132] [django.request:182] [base:get_response] [WARNING]- Not Found: /fl_kt/static/js/kindeditor-4.1.10/kindeditor-min.js
2017-06-26 20:57:53,601 [Thread-11:9000] [django.request:182] [base:get_response] [WARNING]- Not Found: /fl_kt/static/js/kindeditor-4.1.10/kindeditor-min.js
2017-06-26 21:25:55,673 [Thread-4:9212] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/kindeditor-4.1.10/kindeditor-min.js
2017-06-26 21:25:55,865 [Thread-7:7084] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/kindeditor-4.1.10/kindeditor-min.js
2017-06-26 21:26:03,522 [Thread-11:7064] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/kindeditor-4.1.10/kindeditor-min.js
2017-06-27 20:15:23,236 [Thread-3:6424] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-27 21:31:46,025 [Thread-25:11216] [django.request:182] [base:get_response] [WARNING]- Not Found: /uploads/avatar/2016/06/10号北京到兰州.PNG
2017-06-27 21:33:25,160 [Thread-31:5060] [django.request:182] [base:get_response] [WARNING]- Not Found: /mypro/fl_kt/uploads/avatar/2017/06
2017-06-27 21:33:40,230 [Thread-37:1220] [django.request:182] [base:get_response] [WARNING]- Not Found: /fl_kt/uploads/avatar/2017/06
2017-06-27 21:33:51,746 [Thread-43:10960] [django.request:182] [base:get_response] [WARNING]- Not Found: /uploads/avatar/2017/06
2017-06-27 21:34:05,551 [Thread-48:6972] [django.request:182] [base:get_response] [WARNING]- Not Found: /uploads/avatar/2017/06/
2017-06-29 21:45:46,538 [Thread-6:7160] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-29 22:02:01,042 [Thread-11:14648] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-30 19:15:01,374 [Thread-3:8736] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-30 20:01:21,177 [Thread-16:116] [django.request:182] [base:get_response] [WARNING]- Not Found: /admin/upload
2017-06-30 21:08:24,253 [Thread-11:7068] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-06-30 21:31:35,908 [Thread-67:3444] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-01 20:16:41,125 [Thread-3:4580] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:16:41,128 [Thread-4:4896] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:16:41,461 [Thread-5:8756] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-01 20:19:11,257 [Thread-3:10076] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:19:11,259 [Thread-7:10012] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:20:06,655 [Thread-9:3892] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:20:06,661 [Thread-10:6216] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:20:59,062 [Thread-15:6092] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:20:59,062 [Thread-16:6948] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:21:25,128 [Thread-23:9808] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:21:25,135 [Thread-22:3276] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:24:10,250 [Thread-36:5112] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:24:10,267 [Thread-37:4288] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:24:15,526 [Thread-40:8812] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:24:15,526 [Thread-41:7448] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-01 20:40:19,187 [Thread-18:2060] [django.request:182] [base:get_response] [WARNING]- Not Found: /jquery.js
2017-07-01 20:40:19,191 [Thread-19:10204] [django.request:182] [base:get_response] [WARNING]- Not Found: /main.js
2017-07-02 16:20:56,947 [Thread-5:14056] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-02 17:57:16,915 [Thread-8:21616] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'index.html', locals())
  File "C:\mypro\flxm\lib\site-packages\django\shortcuts.py", line 67, in render
    template_name, context, request=request, using=using)
  File "C:\mypro\flxm\lib\site-packages\django\template\loader.py", line 96, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\mypro\flxm\lib\site-packages\django\template\loader.py", line 32, in get_template
    return engine.get_template(template_name, dirs)
  File "C:\mypro\flxm\lib\site-packages\django\template\backends\django.py", line 40, in get_template
    return Template(self.engine.get_template(template_name, dirs), self)
  File "C:\mypro\flxm\lib\site-packages\django\template\engine.py", line 190, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "C:\mypro\flxm\lib\site-packages\django\template\engine.py", line 157, in find_template
    name, template_dirs=dirs, skip=skip,
  File "C:\mypro\flxm\lib\site-packages\django\template\loaders\base.py", line 40, in get_template
    contents = self.get_contents(origin)
  File "C:\mypro\flxm\lib\site-packages\django\template\loaders\filesystem.py", line 25, in get_contents
    return fp.read()
  File "C:\mypro\flxm\lib\codecs.py", line 302, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xcd in position 65: invalid continuation byte
2017-07-02 17:58:18,555 [Thread-10:23144] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/base.css
2017-07-02 17:58:18,559 [Thread-11:23148] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/index.css
2017-07-02 17:58:18,559 [Thread-12:23152] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/jquery.min.js
2017-07-02 17:58:18,563 [Thread-13:23156] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 17:58:18,565 [Thread-14:23160] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 17:58:18,582 [Thread-15:23216] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 17:58:18,684 [Thread-16:23220] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 17:58:18,730 [Thread-17:23224] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 17:58:18,766 [Thread-18:23228] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 17:58:18,806 [Thread-19:23232] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 17:58:18,838 [Thread-20:23236] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 17:58:18,846 [Thread-21:23240] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 17:58:18,858 [Thread-22:23244] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 17:58:18,868 [Thread-23:23248] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 17:58:18,878 [Thread-24:23252] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 17:58:18,884 [Thread-25:23256] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 17:58:19,009 [Thread-26:23264] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 17:58:19,025 [Thread-27:23268] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 17:58:19,032 [Thread-28:23272] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:00:54,920 [Thread-30:23220] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:00:54,930 [Thread-31:23224] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/jquery.min.js
2017-07-02 18:00:54,983 [Thread-32:23228] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/index.css
2017-07-02 18:00:55,009 [Thread-33:23232] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/base.css
2017-07-02 18:00:55,012 [Thread-34:23236] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:00:55,013 [Thread-35:23240] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:00:55,107 [Thread-36:23244] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:00:55,131 [Thread-37:23248] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:00:55,177 [Thread-38:23252] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:00:55,220 [Thread-39:23256] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:00:55,255 [Thread-40:23264] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:00:55,338 [Thread-41:23268] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:00:55,368 [Thread-42:23272] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:00:55,378 [Thread-43:22972] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:00:55,394 [Thread-44:22852] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:00:55,436 [Thread-45:23280] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:00:55,443 [Thread-46:23284] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:00:55,450 [Thread-47:23276] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:00:55,855 [Thread-48:22892] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:00:55,951 [Thread-49:22256] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:00:55,970 [Thread-50:22772] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:00:55,989 [Thread-51:22500] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:00:56,016 [Thread-52:22504] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:00:56,025 [Thread-53:22524] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:00:56,030 [Thread-54:19332] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:00:56,039 [Thread-55:16208] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:00:56,085 [Thread-56:16184] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:00:56,167 [Thread-57:22484] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:00:56,256 [Thread-58:22480] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:01:54,453 [Thread-60:17632] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/base.css
2017-07-02 18:01:54,473 [Thread-61:21216] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/index.css
2017-07-02 18:01:54,476 [Thread-62:6672] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/jquery.min.js
2017-07-02 18:01:54,479 [Thread-63:21532] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:01:54,480 [Thread-64:10532] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:01:54,493 [Thread-65:23356] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:01:54,588 [Thread-66:21964] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:01:54,608 [Thread-67:23444] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:01:54,630 [Thread-68:8184] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:01:54,664 [Thread-69:21852] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:01:54,673 [Thread-70:22280] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:01:54,687 [Thread-71:21424] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:01:54,691 [Thread-72:23500] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:01:54,694 [Thread-73:23496] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:01:54,700 [Thread-74:4432] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:01:54,786 [Thread-75:21264] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:01:54,823 [Thread-76:9480] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:01:54,838 [Thread-77:9728] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:01:54,845 [Thread-78:8156] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:01:54,884 [Thread-79:6564] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:01:54,900 [Thread-80:20740] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:01:54,927 [Thread-81:14340] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:01:57,296 [Thread-83:21600] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/base.css
2017-07-02 18:01:57,299 [Thread-84:23464] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/index.css
2017-07-02 18:01:57,309 [Thread-86:22264] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:01:57,311 [Thread-87:22820] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:01:57,309 [Thread-85:13920] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/jquery.min.js
2017-07-02 18:01:57,322 [Thread-88:21640] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:01:57,411 [Thread-89:22812] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:01:57,414 [Thread-90:22496] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:01:57,505 [Thread-91:21184] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:01:57,507 [Thread-92:21568] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:01:57,545 [Thread-93:22464] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:01:57,552 [Thread-94:22680] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:01:57,605 [Thread-95:22036] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:01:57,609 [Thread-96:21872] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:01:57,611 [Thread-97:17156] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:01:57,618 [Thread-98:20592] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:01:57,622 [Thread-99:20552] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:01:57,627 [Thread-100:20700] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:01:57,714 [Thread-101:20876] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:01:57,724 [Thread-102:22172] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:01:57,730 [Thread-103:22180] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:01:57,763 [Thread-104:20800] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:01:57,769 [Thread-105:21936] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:01:57,775 [Thread-106:21732] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:01:57,869 [Thread-107:18460] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:01:57,888 [Thread-108:22168] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:01:57,894 [Thread-109:19032] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:01:57,908 [Thread-110:23332] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:01:57,921 [Thread-111:17728] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:05:45,660 [Thread-113:9480] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/base.css
2017-07-02 18:05:45,661 [Thread-114:21216] [django.request:182] [base:get_response] [WARNING]- Not Found: /css/index.css
2017-07-02 18:05:45,667 [Thread-115:6564] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/jquery.min.js
2017-07-02 18:05:45,668 [Thread-116:22304] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:05:45,697 [Thread-117:17632] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:05:45,710 [Thread-118:21424] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:05:45,832 [Thread-119:23500] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/sliders.js
2017-07-02 18:05:45,861 [Thread-120:17156] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:05:45,881 [Thread-121:21936] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/jquery.js
2017-07-02 18:05:45,894 [Thread-122:22180] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:05:45,937 [Thread-123:13920] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/up/js.js
2017-07-02 18:05:45,941 [Thread-124:5860] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:05:46,002 [Thread-125:20276] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:05:46,013 [Thread-126:23364] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:05:46,032 [Thread-127:22268] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:05:46,043 [Thread-128:20700] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:05:46,065 [Thread-129:22820] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:05:46,078 [Thread-130:22208] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:05:46,141 [Thread-131:6976] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:05:46,164 [Thread-132:22168] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:05:46,217 [Thread-133:16972] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:05:46,236 [Thread-134:21308] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:05:46,269 [Thread-135:21600] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:05:46,276 [Thread-136:17348] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:05:46,345 [Thread-137:23344] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:05:46,372 [Thread-138:20552] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:05:46,398 [Thread-139:20976] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:05:46,447 [Thread-140:22256] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:05:46,477 [Thread-141:17568] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:09:43,697 [Thread-148:23252] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:09:43,713 [Thread-149:19088] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:09:43,730 [Thread-150:23264] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:09:43,749 [Thread-151:9480] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:09:43,763 [Thread-152:22168] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:09:43,776 [Thread-153:23160] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:09:43,789 [Thread-154:18404] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:09:43,805 [Thread-155:4432] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:09:43,822 [Thread-156:22908] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:09:43,839 [Thread-157:21560] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:09:43,882 [Thread-159:23364] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:09:43,885 [Thread-160:22264] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:09:43,890 [Thread-161:22268] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:09:43,895 [Thread-162:20700] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:09:43,904 [Thread-163:22760] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:09:43,980 [Thread-164:22904] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:09:43,996 [Thread-165:20272] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:09:44,003 [Thread-166:23276] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:09:44,012 [Thread-167:6672] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:09:44,015 [Thread-168:22980] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:09:44,020 [Thread-169:22536] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:09:44,051 [Thread-170:23012] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:09:44,111 [Thread-171:20960] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:09:44,118 [Thread-172:22364] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:16:16,089 [Thread-189:24464] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:16:16,121 [Thread-190:24468] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:16:16,141 [Thread-191:24472] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:16:16,163 [Thread-192:24476] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:16:16,184 [Thread-193:24484] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:16:16,226 [Thread-194:24488] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:16:16,236 [Thread-195:24492] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:16:16,240 [Thread-196:24496] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:16:16,246 [Thread-197:24500] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:16:16,253 [Thread-198:24504] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:16:16,259 [Thread-199:24508] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:16:16,362 [Thread-200:24512] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:16:16,388 [Thread-201:24516] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:16:58,502 [Thread-218:23824] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:16:58,519 [Thread-219:23580] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:16:58,535 [Thread-220:23576] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:16:58,552 [Thread-221:23608] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:16:58,569 [Thread-222:23912] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:16:58,585 [Thread-223:23900] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:16:58,599 [Thread-224:23968] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:16:58,614 [Thread-225:23560] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:16:58,630 [Thread-226:23728] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:16:58,654 [Thread-227:23696] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:16:58,657 [Thread-228:23704] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:16:58,661 [Thread-229:23700] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:16:58,674 [Thread-230:23708] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:16:58,940 [Thread-245:24004] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:16:58,947 [Thread-246:22488] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:17:33,361 [Thread-249:23616] [django.request:182] [base:get_response] [WARNING]- Not Found: /js/nav.js
2017-07-02 18:17:33,391 [Thread-250:23620] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/logo.png
2017-07-02 18:17:33,410 [Thread-251:23640] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:17:33,430 [Thread-252:23624] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:17:33,459 [Thread-253:23604] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:17:33,484 [Thread-254:23600] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:17:33,513 [Thread-255:24308] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:17:33,540 [Thread-256:24320] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:17:33,565 [Thread-257:24232] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:17:33,572 [Thread-258:24272] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:17:33,575 [Thread-259:24184] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:17:33,581 [Thread-260:24168] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:17:33,585 [Thread-261:24092] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:17:33,813 [Thread-272:22264] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:19:33,621 [Thread-281:21668] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:19:33,648 [Thread-282:20700] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:19:33,671 [Thread-283:22216] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:19:33,700 [Thread-284:23772] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:19:33,717 [Thread-285:22736] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:19:33,733 [Thread-286:22740] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:19:33,750 [Thread-287:22884] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:19:33,776 [Thread-288:24408] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 18:19:33,779 [Thread-289:21380] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/05.jpg
2017-07-02 18:19:33,785 [Thread-290:24044] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 18:19:33,789 [Thread-291:24148] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/ad300x100.jpg
2017-07-02 18:19:34,048 [Thread-306:20676] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 18:19:34,071 [Thread-307:22588] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 18:19:34,092 [Thread-308:17796] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:24:06,358 [Thread-311:14100] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a1.jpg
2017-07-02 18:24:06,375 [Thread-312:22588] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:24:06,400 [Thread-313:23528] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:24:06,417 [Thread-314:24404] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:24:06,683 [Thread-322:24004] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:25:46,211 [Thread-340:21360] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a2.jpg
2017-07-02 18:25:46,232 [Thread-341:23760] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a3.jpg
2017-07-02 18:25:46,296 [Thread-342:24296] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/a4.jpg
2017-07-02 18:25:46,323 [Thread-343:23908] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:25:46,648 [Thread-358:23760] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:10,404 [Thread-365:24996] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:29,382 [Thread-383:25160] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:29,713 [Thread-398:25224] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:32,875 [Thread-401:25252] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:33,236 [Thread-416:25312] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:35,454 [Thread-419:25356] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:35,750 [Thread-434:25420] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:36,799 [Thread-437:25448] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:37,124 [Thread-452:25508] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:38,460 [Thread-455:25532] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:38,818 [Thread-470:25596] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:28:41,510 [Thread-471:21040] [django.request:182] [base:get_response] [WARNING]- Not Found: /news.html
2017-07-02 18:28:59,801 [Thread-473:20300] [django.request:182] [base:get_response] [WARNING]- Not Found: /c.html
2017-07-02 18:29:10,762 [Thread-476:24632] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:29:11,075 [Thread-491:25000] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:29:12,865 [Thread-494:25020] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:29:13,201 [Thread-509:24908] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:33:08,420 [Thread-512:25572] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:33:09,099 [Thread-527:25332] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:33:12,982 [Thread-530:24956] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:33:13,282 [Thread-545:25120] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:34:09,457 [Thread-548:25440] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:34:09,736 [Thread-563:24928] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:36:24,207 [Thread-566:20816] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 18:36:24,674 [Thread-581:25552] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:19:43,678 [Thread-584:23700] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:19:44,078 [Thread-599:26680] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:19:54,874 [Thread-602:29008] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:19:55,486 [Thread-617:27492] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:59:19,635 [Thread-767:31708] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 20:59:19,670 [Thread-768:30468] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 20:59:19,700 [Thread-769:30952] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:59:19,739 [Thread-770:31668] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 20:59:19,776 [Thread-771:31672] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/04.jpg
2017-07-02 20:59:20,082 [Thread-786:31120] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/01.jpg
2017-07-02 20:59:20,092 [Thread-787:31092] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/02.jpg
2017-07-02 20:59:20,111 [Thread-788:16348] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/03.jpg
2017-07-02 20:59:20,125 [Thread-789:28236] [django.request:182] [base:get_response] [WARNING]- Not Found: /images/06.jpg
2017-07-02 21:09:27,759 [Thread-857:9280] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-02 21:09:33,519 [Thread-858:10424] [django.request:182] [base:get_response] [WARNING]- Not Found: /amdin
2017-07-03 19:36:06,345 [Thread-19:26388] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-03 19:56:26,500 [Thread-345:31592] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-03 21:01:36,482 [Thread-2:27052] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'index.html', {'yy_list': yy_list})
UnboundLocalError: local variable 'yy_list' referenced before assignment
2017-07-03 21:10:48,410 [Thread-8:29376] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'index.html', {'yy_list': yy_list})
UnboundLocalError: local variable 'yy_list' referenced before assignment
2017-07-03 21:13:20,084 [Thread-2:26712] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'base.html', {'yy_list': yy_list})
UnboundLocalError: local variable 'yy_list' referenced before assignment
2017-07-03 21:16:29,500 [Thread-2:8460] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'index.html', {'yy_list': yy_list})
UnboundLocalError: local variable 'yy_list' referenced before assignment
2017-07-03 21:16:35,447 [Thread-3:2556] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'index.html', {'yy_list': yy_list})
UnboundLocalError: local variable 'yy_list' referenced before assignment
2017-07-03 21:16:51,903 [Thread-6:31988] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\fl_kt\Acc\views.py", line 21, in index
    return render(request, 'index.html', {'yy_list': yy_list})
UnboundLocalError: local variable 'yy_list' referenced before assignment
2017-07-04 20:45:18,789 [Thread-20:17004] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-04 20:45:38,033 [Thread-42:32096] [django.request:182] [base:get_response] [WARNING]- Not Found: /favicon.ico
2017-07-04 20:54:09,723 [Thread-6:27356] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 20:55:48,953 [Thread-18:13140] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:03:00,467 [Thread-3:18164] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:04:13,446 [Thread-8:26220] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:04:32,545 [Thread-21:380] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:05:21,668 [Thread-3:14712] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (RZSJJ) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:06:34,503 [Thread-3:26924] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (kaishi) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:23:39,134 [Thread-54:29864] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (kaishi) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:24:20,180 [Thread-2:16672] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:24:45,901 [Thread-2:6064] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:27:03,171 [Thread-2:16068] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:28:08,927 [Thread-8:10164] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:32:58,940 [Thread-2:27144] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (bz1) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
2017-07-04 21:35:53,891 [Thread-2:20464] [django.request:284] [base:handle_uncaught_exception] [ERROR]- Internal Server Error: /admin/Acc/rzxx/add/
Traceback (most recent call last):
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\mypro\flxm\lib\site-packages\django\core\handlers\base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 541, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\sites.py", line 244, in inner
    return view(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1437, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\mypro\flxm\lib\site-packages\django\utils\decorators.py", line 184, in inner
    return func(*args, **kwargs)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 1367, in changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\mypro\flxm\lib\site-packages\django\contrib\admin\options.py", line 642, in get_form
    % (e, self.__class__.__name__))
FieldError: Unknown field(s) (rzsj) specified for Rzxx. Check fields/fieldsets/exclude attributes of class RzxxAdmin.
